docs/concepts/portal-loop.md

```diff
@@ -67,3 +67,15 @@ has some drawbacks:
 Gno will fail to be replayed, meaning **data will be lost**. 
 - Since transactions are archived and replayed during genesis, 
 block height & timestamp cannot be relied upon.
+
+### Permissionless deployments to Portal Loop
```

Comment:
```suggestion
### Deploying to the Portal Loop
```

Let's use this paragraph to talk about deploying to the portal loop in general. Let's mention permissionless deployment, as well as build upon the previous paragraphs to tell people that, essentially, `master` contains the `examples/` folder and is the first to be loaded into genesis. Technically speaking this also means upgradeability for `examples/` realms & packages, and people should be aware of that.

Reply:
Rephrased the paragraph :)

Reply:
wait. not good yet

Reply:
ok. Please tell me what to change.